cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -775,13 +775,13 @@ define([
 
         expectGroupsVisible = function(view, groupNames) {
             _.each(groupNames, function(groupName) {
-                expect(view.$('.content-groups-list-item')).toContainText(groupName);
+                expect(view.$('.partition-groups-list-item')).toContainText(groupName);
             });
         };
 
         beforeEach(function() {
             TemplateHelpers.installTemplates(
-                ['content-group-editor', 'content-group-details', 'list']
+                ['content-group-editor', 'configuration-group-details', 'list']
```

Comment:
We discussed this in person.  Please let me know if you would still like more changes to this.